test/index.js

```diff
@@ -100,8 +101,58 @@ function testRun(i) {
 		testIt(pass2028, new Buffer('2028 rsa key with variant passwords'), t);
 	});
 }
+
+
+// Test RSA encryption/decryption
+test('node tests', function (t) {
+	var certPem = fs.readFileSync(__dirname + '/test_cert.pem', 'ascii');
+	var keyPem = fs.readFileSync(__dirname + '/test_key.pem', 'ascii');
+	var rsaPubPem = fs.readFileSync(__dirname + '/test_rsa_pubkey.pem',
+	    'ascii');
+	var rsaKeyPem = fs.readFileSync(__dirname + '/test_rsa_privkey.pem',
+	    'ascii');
+	var rsaKeyPemEncrypted = fs.readFileSync(
+  __dirname + '/test_rsa_privkey_encrypted.pem', 'ascii');
+  var input = 'I AM THE WALRUS';
+  var bufferToEncrypt = new Buffer(input);
+
+  var encryptedBuffer = myCrypto.publicEncrypt(rsaPubPem, bufferToEncrypt);
+
+  var decryptedBuffer = myCrypto.privateDecrypt(rsaKeyPem, encryptedBuffer);
+  t.equal(input, decryptedBuffer.toString());
+
+  var decryptedBufferWithPassword = myCrypto.privateDecrypt({
+    key: rsaKeyPemEncrypted,
+    passphrase: 'password'
+  }, encryptedBuffer);
+  t.equal(input, decryptedBufferWithPassword.toString());
+
+  // encryptedBuffer = myCrypto.publicEncrypt(certPem, bufferToEncrypt);
```

Comment:
this is a test from io.js but I haven't implemented certificates. so I hope to be able to uncomment that so we can pass all of the io.js tests.